packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -508,6 +509,8 @@ const defaultCompressionConfig = {
 
 const defaultChunkSizeInBytes = 204800;
 
+export const defaultCloseSummarizerDelayMs = 10000; // 10 seconds
```

Comment:
Looks like this belongs in the `runnningSummarizer` and should be exporeted here.

Reply:
Not exactly sure the difference between one and the other, we already import the containerRuntime file in runningSummarizer, so I was concerned about any build breaks that might potentially happen because of that. I don't know if I should be doing tech debt in this PR.